accepted/targeting-packs/targeting-packs.md

```diff
@@ -0,0 +1,149 @@
+installed on the developer's machine. This allows developers to target a lower
+version than what they use themselves. This is especially useful for in-place
+updates where the older version of the framework cannot be installed side-by-
+side.
+
+That mechanism is called *targeting packs*, which is simply a set of assemblies
+that describe the public APIs of a particular version of the .NET Framework.
+Since those assemblies are only used by the compiler, they don't need to contain
+any code, which we call [reference assemblies](https://www.youtube.com/watch?v=EBpY1UMHDY8).
+
+Unfortunately, the targeting packs aren't available by themselves; they are
+redistributed as part of the .NET Framework Developer Packs which bundle both
+the runtime as well as the targeting pack. This design ensures that developers
+can never be in a state where they have a targeting pack that is higher than the
+.NET Framework version they have installed. That design made sense coming from a
+Visual Studio-first perspective but doesn't serve us well today:
+
+* **It's not cross-platform friendly**. The .NET Framework Developer Packs are
+  only available for Windows which makes the targeting packs indirectly Windows-
+  only too.
+
+* **It's not CI friendly**. Requiring a machine-wide install isn't friendly for
+  cloud-hosted CI machines as developers typically lack permissions to install
+  software.
+
+This document proposes to offer the targeting packs via NuGet that they can be
+acquired in a non-impactful way. It builds on the following existing specs:
+
+* [Phillip Carter](https://github.com/cartermp): [Add Target Packs on NuGet](https://github.com/dotnet/core/pull/64)
+* [Daniel Plaisted](https://github.com/dsplaisted): [Reference Assembly NuGet packages](https://gist.github.com/dsplaisted/83d67bbcff9ec1d0aff1bea1bf4ad79a)
+* [Prototype](https://github.com/dsplaisted/ReferenceAssemblyPackages)
+
+## Scenarios and User Experience
+
+* Build a project which is multi-targeted to .NET Core or .NET Standard as well
+  as .NET Framework on Mac OS or Linux using the .NET CLI
+
+* Build a project targeting Mono (which uses the same Target Framework
+  Identifier as .NET Framework) using the .NET CLI
+
+## Requirements
+
+### Goals
+
+* Doesn't require running an installer
+* Doesn't require Windows
+* Can be used in SDK-style multi-targeting projects
+
+### Non-Goals
+
+* Supporting NuGet-based acquisition of the .NET Framework runtime
+* Supporting NuGet-based acquisition from non-SDK-style projects
+* Support pre-.NET Framework 4.5 targeting packs
+* Support non-.NET Framework targeting packs (Xamarin, Mono, Unity)
+
+## Design
+
+* Daniel has created a [prototype](https://github.com/dsplaisted/ReferenceAssemblyPackages).
+    - We have an identity package
+    - Each version has a different identity and is looped in by TFM
+    - Ensures only incremental download
+    - Open question is whether we need the aggregate package at all or just
```

Comment:
The aggregate package helps enable the scenario when you don't use `Microsoft.NET.Sdk`, since there's just one package you reference.  From the SDK, we may not end up referencing the aggregate package, as then we could just reference version 1.0.0 by default of the version-specific package unless we had to service one of the packages.

Reply:
Got it. Will reword.